### DISCUSSION TITLE
Redux Toolkit with Next.js 13 features

### DISCUSSION THREAD

**Discussion by its-monotype** [vercel/next.js#42224] [Help]
How to use Redux Toolkit with Next.js 13 with new fetching and new routing?

**Comment by tonypizzicato** [comment]
at least, how to setup the store to be shown in devtools. right now i see the next-router state. no access to my app's store
<img width="1590" alt="Screenshot 2022-11-04 at 20 05 31" src="https://user-images.githubusercontent.com/640122/200055711-ab9d255b-9b11-480d-ae57-3256e1efade4.png">

**Reply by mezhinsky** [reply]
same! the DevTools connection only after you make some changes in state
